packages/components/src/font-size-picker/utils.ts

```diff
@@ -44,3 +44,20 @@ export function getCommonSizeUnit( fontSizes: FontSize[] ) {
 	} );
 	return areAllSizesSameUnit ? firstUnit : null;
 }
+
+/**
+ * hasValidShortName
+ *
+ * check that the font size short name is a string with at most
+ * 3 characters length (e.g. "XXS", "XS", "S", "M", "L", "XL", "XXL")
+ *
+ * @param  shortName font size object
+ * @return boolean
+ */
+export function hasValidShortName( shortName: string | undefined ): boolean {
+	return (
+		typeof shortName === 'string' &&
```

Comment:
Is this check necessary? Is it possible to pass e.g. a number to `shortName`? I thought `theme.json` was validated against the schema but could be wrong.

Reply:
I can't see any coercion going on in the background. In fact, without the type check it'll throw errors if `shortName` is `0` or `undefined`:

<img width="568" alt="Screen Shot 2022-12-16 at 1 43 14 pm" src="https://user-images.githubusercontent.com/6458278/208010156-adb733a3-efb8-464e-a31e-0e33bef3ca7e.png">

So it seems good to leave it as is?